src/movies.js

```diff
@@ -1,22 +1,80 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(moviesArray) {
+  let directores = moviesArray.map((directors) => {
+    return directors.director
+  })
+  return directores
+}
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
-function howManyMovies(moviesArray) {}
+function howManyMovies(moviesArray) {
+  let total = 0
+  moviesArray.filter((numSpielberg) => {
```

Comment:
- Podrías utilizar un AND (&&) dentro del if en lugar de el ciclo for para ahorrarte código.
- Contando los elementos con total++ se pierde un poco el sentido de usar .filter.